docs/_sources/studios/airwaze/index.rst.txt

```diff
@@ -53,7 +47,9 @@ Also create a database for your tests::
 
 Enable Geospatial Extensions in the Airwaze Database
 =====================================================
-Now we want to install the geospatial extensions to Postgres for the ``airwaze`` db. Open the Postgres UI and double click on the ``airwaze`` db to open a ``psql`` command prompt connected to the ``airwaze`` db::
+Now we want to install the geospatial extensions to Postgres for the ``airwaze`` db. 
+
+From the PSQL CLI run ``\c airwaze`` to connect to the airwaze database::
```

Comment:
You could move this into the code block below and add comments as to what it does